### ISSUE TITLE
scala autotranslation error on Windows

### ISSUE THREAD

**Issue by EfimovVladimir** [twosigma/beakerx#7589]
We can see the error on test/ipynb/groovy/AutotranslationBetweenKernels.ipynb notebook.
![image](https://user-images.githubusercontent.com/17744188/41971533-763159a6-7a17-11e8-9923-45b6de04c9d3.png)

terminal log:
```
[I 14:38:16.792 NotebookApp] Kernel started: f549a927-969a-4d61-aa17-14346899157f
[I 14:38:18.962 NotebookApp] Adapting to protocol v5.1 for kernel f549a927-969a-4d61-aa17-14346899157f
Exception in thread "main" java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.String
        at com.twosigma.beakerx.AutotranslationServiceImpl.<init>(AutotranslationServiceImpl.java:54)
        at com.twosigma.beakerx.AutotranslationServiceImpl.createAsSubkernel(AutotranslationServiceImpl.java:38)
        at com.twosigma.beakerx.NamespaceClient.create(NamespaceClient.java:182)

        at com.twosigma.beakerx.scala.kernel.Scala.lambda$main$0(Scala.java:82)
        at com.twosigma.beakerx.kernel.KernelRunner.run(KernelRunner.java:24)
        at com.twosigma.beakerx.scala.kernel.Scala.main(Scala.java:78)
Traceback (most recent call last):
  File "C:\Aconda3\envs\beakerx\Scripts\beakerx-script.py", line 11, in <module>
    load_entry_point('beakerx', 'console_scripts', 'beakerx')()
  File "d:\pr\twosigma\beakerx\beakerx\beakerx\__init__.py", line 46, in run  parse()
  File "d:\pr\twosigma\beakerx\beakerx\beakerx\commands.py", line 91, in parse  args.func(args)
  File "d:\pr\twosigma\beakerx\beakerx\beakerx\commands.py", line 65, in start_py4j_server
    Py4JServer(args.port, args.pyport, args.kernel, args.context)
  File "d:\pr\twosigma\beakerx\beakerx\beakerx_magics\jvm_kernel_magic.py", line 101, in __init__
    pep = PythonEntryPoint(kernel_name, context)
  File "d:\pr\twosigma\beakerx\beakerx\beakerx_magics\jvm_kernel_magic.py", line 71, in __init__
    self.pm = JVMKernelMagic(kernel_name, context)
  File "d:\pr\twosigma\beakerx\beakerx\beakerx_magics\jvm_kernel_magic.py", line 30, in __init__
    self.start()
  File "d:\pr\twosigma\beakerx\beakerx\beakerx_magics\jvm_kernel_magic.py", line 38, in start
    self.kc.wait_for_ready()
  File "C:\Aconda3\envs\beakerx\lib\site-packages\jupyter_client\blocking\client.py", line 120, in wait_for_ready
    raise RuntimeError('Kernel died before replying to kernel_info')
RuntimeError: Kernel died before replying to kernel_info
```

**Comment by lmitusinski** [comment]
It has been fixed with #7617, please do retest.

**Comment by EfimovVladimir** [comment]
fixed
